sandbox/rocky/tf/algos/ddpg/ddpg.py

```diff
@@ -0,0 +1,233 @@
+from sandbox.rocky.tf.algos.network.actor_critic_net import ActorNet, CriticNet
+from sandbox.rocky.tf.algos.ddpg.replay_buffer import ReplayBuffer
+from rllab.algos.base import RLAlgorithm
+
+import tensorflow as tf
+from copy import copy
+import numpy as np
+import os
+
+
+class DDPG(RLAlgorithm):
+    def __init__(self,
+                 env,
+                 gamma=0.99,
+                 tau=0.001,
+                 observation_range=(-5, 5),
+                 action_range=(-1, 1),
+                 actor_lr=1e-4,
+                 critic_lr=1e-3,
+                 reward_scale=1,
```

Comment:
`reward_scale` should be called `discount` (in accordance with the other implementations). A value of 1 seems strange, are you sure it isn't <1, e.g. 0.99?

Reply:
In the paper and baseline, they all set this value to be 1.